packages/kbn-telemetry-tools/src/tools/serializer.ts

```diff
@@ -104,6 +109,25 @@ export function getConstraints(node: ts.Node, program: ts.Program): any {
     return getConstraints(declarationNode, program);
   }
 
+  // node input ( enum { A = 'my_a', B = 'my_b' } ). returns ['my_a', 'my_b'];
+  if (ts.isEnumDeclaration(node)) {
+    const { parent, ...rest } = node;
```

Comment:
NIT: why are we extracting `parent` here? I can't see a place where we are using it 🤔

Reply:
@afharo oh! i `console.log(rest)` for quick debugging since parent is a huge object that i am not interested in!